rainbow.py

```diff
@@ -114,26 +115,28 @@ def ProcessImage(self, image, colour, screen):
         # Green is between 50 and 75
         # Blue is between 20 and 35
         # Yellow is... to be found!
-        if colour == "red":
-            imrange = cv2.inRange(image, numpy.array((95, 127, 64)), numpy.array((125, 255, 255)))
-        elif colour == "green":
-            imrange = cv2.inRange(image, numpy.array((50, 127, 64)), numpy.array((75, 255, 255)))
-        elif colour == 'blue':
-            imrange = cv2.inRange(image, numpy.array((20, 64, 64)), numpy.array((35, 255, 255)))
- 
+        colour_bounds = {
+            'red': ((95, 127, 64), (125, 255, 255)),
+            'green': ((50, 127, 64), (75, 255, 255)),
+            'blue': ((20, 64, 64), (35, 255, 255)),
+            'yellow': ((20, 64, 64), (35, 255, 255)),
```

Comment:
Note: I have just set the yellow lower/upper bounds to be the same as that of blue (as you'll be updating it anyhow)